### PR TITLE
Update login screens

### PR DESCRIPTION
**Pull request by davidyuk**
This PR depends on an unpublished version of `aepp-components` and https://github.com/aeternity/aepp-components/pull/37.

### CODE REVIEW

**Comment by emil-apeunit** [comment]
dependent PR is merged

**Comment by davidyuk** [comment]
Seems that I have a wording problem.

**Comment by dadaphl** [comment]
I get several of these messages, for example after clicking `Login` Button. Not sure if its a bug in components, though.
```
[Vue warn]: Invalid prop: custom validator check failed for prop "name".
found in

---> <AeIcon> at src/components/aeIcon/aeIcon.vue
       <AeButton> at src/components/aeButton/aeButton.vue
         <AeOverlay> at src/components/aeOverlay/aeOverlay.vue
           <AeModal> at src/components/aeModal/aeModal.vue
             <ModalScreen> at src/components/ModalScreen.vue
               <NewAccount> at src/pages/NewAccount.vue
                 <App> at src/App.vue
                   <Root>
```

**Comment by dadaphl** [comment]
please talk to Shaliah about wording issues. Also the input field is jumping while typing. Maybe thats a component issue.
screen shot 2018-02-01 at 17 13 22
screen shot 2018-02-01 at 17 13 18

**Comment by dadaphl** [comment]
I think it would be better if the button reads `Setup` as im starting the app for the first time.
screen shot 2018-02-01 at 18 45 32

**Comment by dadaphl** [comment]
Font color, line height and button size (width) are different than in screen before. Creates a jumpy/buggy effect

Also headlines (next to the X button) are not centered

screen shot 2018-02-01 at 18 46 38

**Comment by dadaphl** [comment]
`See how it works` leeds to new Account screen. Should there be a on-boarding screen?

**Comment by dadaphl** [comment]
Notification style is not correct
**is:**
screen shot 2018-02-01 at 18 50 48
**should:**
screen shot 2018-02-01 at 18 51 25

File: src/pages/Apps/Apps.scss [hunk 1]
```diff
@@ -1,13 +1,5 @@
...
-  .nomargin {
-    visibility: hidden;
-    &.visible {
-      visibility: visible;
-    }
-    .ae-notification {
-      margin-bottom: 0;
-    }
-  }
+  margin-top: 65px;
```
[the remark this answers is not in the document]
Comment (dadaphl) [reply src/pages/Apps/Apps.scss]: please also see #39

**Comment by davidyuk** [comment]
@dadaphl  `Invalid prop: custom validator check failed for prop "name"` in AeIcon happen because this PR depends on an unpublished version of `aepp-components` package.
Can you publish `aepp-components` with latest features?
Yes, input fields a jumping because help text is moved to a new line. I have made a PR that fixes it https://github.com/aeternity/aepp-components/pull/38.
Onboarding screens are not merged yet, so I can't point to them from Intro page.

**Pull request merged**